dogs.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Our Dogs</title>
+    <script src="js/dogs-javascript.js" charset="utf-8"></script>
+    <link href="styles/stylesheet.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <section class="hdr-left">
+            <img class="logo" src="images/adoptadog-logo.svg" alt="Adopt a dog logo">
+            <h1>Seattle's premiere dog adoption agency</h1>
+        </section> <!-- end of hdr-left -->
+
+        <section class="hdr-right">
+            <div class="cart">
+                <img class="cart" src="images/cart-icon.svg" alt="image of a cart">
+                <p>$1234.56</p>
+            </div>
+            <!--end of cart  -->
+            <button type="submit" value="Submit" name="submit">Checkout</button>
+        </section> <!-- end of hdr-right -->
+
+
+    </header>
+
+    <nav>
+        <ul>
+            <li> <a href="./index.html">Home</a></li>
+            <li>/</li>
+            <li> <a href="./dogs.html">Dogs</a></li>
+            <li>/</li>
+            <li> <a href="./blog.html">Blog</a></li>
+
+        </ul>
+    </nav>
+    <div class="wrapper">
+        <h2>Here are all of our Dogs!</h2>
+
+        <section class="all-dogs">
+
+            <article class="col-1 row-1">
+                <img src="images/murphy-card.jpg" alt="Tan and white boxer-mix named Murphy"
+                    onclick=" dogInfo('Murphy', 'Boxer Mix', '$123.45.')">
+                <h3>
+                    Murphy
+                </h3>
+                <h4>
+                    Cost to adopt: $123.45
+                </h4>
+                <p>
+                    Nullam at urna convallis, ornare turpis ut, fermentum odio. Sed auctor non justo ut imperdiet.
+                    Maecenas
+                    feugiat facilisis mi nec vehicula.
+                </p>
+                <button type="submit" value="Adopt" name="adopt" onclick="sumCost(123.45)">Adopt</button>
+        </article>
+
+        <article class = " col-2 row-1">
+                    <img src="images/poppy-card.jpg" alt="Gray and white Shih-tzu named Poppy"
```

Comment:
Your indenting makes it a bit hard to read. Most code editors will have a shortcut to ident for you, but there is also websites like this one you can run your code through to make it easier to read:  https://www.freeformatter.com/html-formatter.html

Doesn't matter whether you use tabs or spaces, but is good practice to indent your code and keep whichever you choose consistent across your code